### DISCUSSION TITLE
Custom fields or columns should have an option to be shared across Projects (Beta)

### DISCUSSION THREAD

**Discussion by cbarina** [community/community#10853] [Projects and Issues] [answered]
Glad to see that you can attach a date field and other dropdowns to boards. 

We wanted to create a public-facing board (with a subset of issues, read-only) and an internal board (with another, possibly overlapping set of issues) as some commentary/tasks are internal. I thought I had sorted this out somewhat by creating two separate projects.

However, custom columns/fields do not seem to be accessible outside their own project - so, a due-by field in "Public board" needs to be recreated and re-entered in the "Internal board". 

Is there a way somehow to make these fields shared across a project, or synced in some way?

Alternatively, if there were view-level permissions/sharing within a project, that would work too - could just make one view for public-facing, other for internal-facing.

**Comment by gawertm** [comment]
@cbarina did you manage to find a solution? Currently trying to achieve the same.  also found out about this one: https://github.com/github/roadmap/issues/277

**Comment by Jay-o-Way** [comment]
What about different _views_ ?

**Reply by 0xSwapFeeder** [reply]
> What about different _views_ ?

Well, when you already have 15 different views for a single use case (lets say project manage with user and technical stories meta issues) you want to be able to have at least a secondary project for tasks and be able to make workflows between those 2 projects.

I mean that's also an issue I have.

**Comment by bargenson** [comment]
Same problem. It's making hard to manage issues across multiple projects without using labels for everything :-\

**Comment by florent-solt-wmx** [comment]
Same here, it's almost impossible to manage multiple projects

**Comment by cailafinn** [comment]
Same issue here. Would be really nice to have estimates carry over between projects.

**Comment by adam-leeper-ai** [comment]
+1. Searched if this was possible and ended up here :)

**Comment by lynwilhelm** [comment]
+1

**Comment by labudis** [comment]
Hi there 👋

I'm a Senior Product Manager for Issues at GitHub, and I wanted to share that my team is in the early stages of exploring a new feature: Issue fields. This would allow you to define fields at the Organization level, set their values once, and have them available across all repositories and Projects—making it much easier to share and standardize issue data.

While this feature is still in the early exploration phase, we understand the need for better cross-Project issue data sharing and are actively working toward a solution. I’ll keep you updated as we make progress!

**Reply by pombredanne** [reply]
This would be much welcomed. At the moment, using issues to track and project-managed seriously any kind of large org with many repos is difficult, error prone and requires tons of manual work or API calls.

**Reply by pombredanne** [reply]
Just to say that the lack of org-level options not only for fields, (including statuses) but also for labels and milestones, as well as simple options to track and report issues across repos makes it painful to use GH issues seriously.

**Reply by jkgerig** [reply]
Has there been any progress on this feature, @labudis ? My organization could really benefit from this and I'd prefer to not invest time in building my own automation if there's a built-in solution coming soon.

**Reply by acolchagoff** [reply]
We built a workflow around multiple projects, each team has a project contains the current iteration for their work, and then there is a bug List project for triage. The problem is, when a bug is triaged, it is added to the appropriate team's project but none of the bug meta comes over with it, because its all custom fields. We are resorting to abusing labels for important metadata but the ability to define one custom field for several projects would be really nice.

**Reply by labudis** [reply]
Hi folks, Issue fields are now in private preview, it should solve most of the problems you are describing in this thread: https://github.com/orgs/community/discussions/10853#discussioncomment-15703003

**Comment by labudis** [comment, marked as answer]
We've released [Issue fields](https://github.com/orgs/community/discussions/175366) in Private Preview which allows Organizations to define metadata Issue fields that span all repositories and Projects within the Organization. It solves the problem of keeping Issue metadata in sync across multiple Projects as well as reducing the reliance on labels.

You can request access by posting a comment in [this discussion](https://github.com/orgs/community/discussions/175366) with your org slug.
